rustfmt.toml

```diff
@@ -0,0 +1 @@
+# defaults are great :)
```

Comment:
If defaults are great, is there a reason this file was added here?

Reply:
I added the reason in the commit message:

> By adding an (empty) `rustfmt.toml` file, you signal to developers and
code editors that `rustfmt` is required, and the default values are
used.

Many (large) projects have not yet adopted Rustfmt, so enabling it by default in your editor – if you work on many different open source projects – can result in unwanted commits/diffs when contributing to those projects.

So instead, what I do is only enable Rustfmt automatically if the editor detects a `rustfmt.toml` file in the root of the project. But aside from the potential for automation, it also signals to developers that this project uses Rustfmt, even though we use all the defaults.

There are [some other projects](https://github.com/search?q=filename%3Arustfmt.toml+defaults) that do the same, although admittedly not as much as I'd like.

Reply:
By the way, at first I also added a `rust-toolchain` file containing `stable` as its contents, as the nightly rustfmt and stable rustfmt behave differently with defaults. That would be another automated signals to tools to use the stable toolchain when working on this project.

Unfortunately, that broke the CI, so I removed it from the commit.